internal/js-dependency-scanner/README.md

```diff
@@ -0,0 +1,48 @@
+## Security tool to automating the scan related to vulnerabilities of Java Script Libraries used in products.
+
+*This tool performs security scans for weekly release of every WSO2 products. The weekly release of products
+downloaded from respective github repository. 
+
+*This tool currently uses Retire.js as a JS Security Scan tool. Retire.js is an open source tool
+which helps to detect JS libraries with known vulnerabilities.
+
+*This tool generate a report for each product which includes retire.js result as json format.
+
+*This tool delivers the by fallowing ways : 
+        1. Upload reports to git repo : 
+        2. If the report contains any known vulnerability, Create issue ticket. Currently it supports JIRA.
+        3. Integrate with VMS.
+        
+* Before you begin : 
+
+    1. Install Oracle Java SE Development Kit (JDK) version 1.7* or 1.8 and set the JAVA_HOME environment variable.
+
+    2. Install Maven
+
+    3. Clone : https://github.com/mathuriga/WSO2-SecurityScanJsLibs.git
+    
```

Comment:
People should not need to clone a personal repo.

Reply:
Resolved.